app/views/users/_form.html.erb

```diff
@@ -9,6 +9,6 @@
   <%= form.text_field :handle, :class => 'form__input' %>
 </div>
 <div class="password_field">
-  <%= form.label :password, :class => 'form__label' %>
+  <%= form.label :password, t('activerecord.attributes.user.password'), :class => 'form__label' %>
```

Comment:
To fix the password label translation this commit manually adds the
translation of the attribute instead of relying on the action-view auto
translation. 

This happens because action-view looks before for a
helpers.label.password translation before trying to use the
human_attribute_name of it 
https://github.com/rails/rails/blob/05dc13f91bf23f6ee9682084865a5e6b794aac45/actionview/lib/action_view/helpers/tags/translator.rb#L14-L17

this leads into the 
I18n::Backend::Simple and Fallback running with
```
    key
    => "user.password"
    locale
    => :"pt-BR"
    options
    => {:default=>[:"user.password", ""], :scope=>"helpers.label"}
    ```
to return `Password`, even we haven't defined the key on the pt-BR translation.

Clearance defines this translation key in
their Gem on english and we fallback to it since we don't have it
defined on pt-BR
https://github.com/thoughtbot/clearance/blob/fbaf5cf368bc6512c11251559c5ecb655276994f/config/locales/clearance.en.yml#L30

We could define
```
helpers:
  label:
    user:
      password: Senha
```
on each translation file, but I preferred to call the translate directly on the view.